src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -356,13 +359,14 @@ public static CloseableHttpClient buildHttpClient(
       connectionManager.setMaxTotal(maxConnections);
       connectionManager.setDefaultMaxPerRoute(maxConnectionsPerRoute);
 
+      String userAgentIdentifier = key != null ? key.getUserAgentIdentifier() : "";
```

Comment:
Rename `getUserAgentIdentifier()` to `getUserAgentSuffix()` everywhere

Reply:
Done